server/config/express.js

```diff
@@ -113,11 +114,6 @@ export default function() {
     })
   })
 
-  // let the client handle it
-  app.use(function(req, res) {
-    res.sendFile(path.resolve('./client/index.html'))
-  })
```

Comment:
Don't we still need this? When I take this out it breaks going directly to URLs for the application pages such as when I paste http://localhost:3000/users/signin into my browser I get not found.